<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>272</width>
                <height>480</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Contactless Reader</string>
        </property>

        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <property name="spacing">
                    <number>0</number>
                </property>
                <property name="leftMargin">
                    <number>0</number>
                </property>
                <property name="topMargin">
                    <number>0</number>
                </property>
                <property name="rightMargin">
                    <number>0</number>
                </property>
                <property name="bottomMargin">
                    <number>0</number>
                </property>
                <item>
                    <widget class="QStackedWidget" name="stackedWidget">

                        <!-- Scan page -->
                        <widget class="QWidget" name="pageScan">
                            <property name="styleSheet">
                                <string notr="true">background-color: #2196F3; color: white;</string>
                            </property>
                            <layout class="QVBoxLayout" name="layoutScan">
                                <property name="leftMargin">
                                    <number>20</number>
                                </property>
                                <property name="topMargin">
                                    <number>20</number>
                                </property>
                                <property name="rightMargin">
                                    <number>20</number>
                                </property>
                                <property name="bottomMargin">
                                    <number>20</number>
                                </property>
                                <item>
                                    <widget class="QLabel" name="labelScan">
                                        <property name="text">
                                            <string>Weka kadi yako hapa</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                        <property name="wordWrap">
                                            <bool>true</bool>
                                        </property>
                                        <property name="font">
                                            <font>
                                                <pointsize>18</pointsize>
                                                <weight>75</weight>
                                                <bold>true</bold>
                                            </font>
                                        </property>
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                                                <horstretch>0</horstretch>
                                                <verstretch>0</verstretch>
                                            </sizepolicy>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </widget>

                        <!-- Processing page -->
                        <widget class="QWidget" name="pageProcessing">
                            <property name="styleSheet">
                                <string notr="true">background-color: #FF9800; color: white;</string>
                            </property>
                            <layout class="QVBoxLayout" name="layoutProcessing">
                                <property name="leftMargin">
                                    <number>20</number>
                                </property>
                                <property name="topMargin">
                                    <number>20</number>
                                </property>
                                <property name="rightMargin">
                                    <number>20</number>
                                </property>
                                <property name="bottomMargin">
                                    <number>20</number>
                                </property>
                                <item>
                                    <widget class="QLabel" name="labelProcessing">
                                        <property name="text">
                                            <string>Inaendelea...</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                        <property name="wordWrap">
                                            <bool>true</bool>
                                        </property>
                                        <property name="font">
                                            <font>
                                                <pointsize>18</pointsize>
                                                <weight>75</weight>
                                                <bold>true</bold>
                                            </font>
                                        </property>
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                                                <horstretch>0</horstretch>
                                                <verstretch>0</verstretch>
                                            </sizepolicy>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </widget>

                        <!-- Error page -->
                        <widget class="QWidget" name="pageError">
                            <property name="styleSheet">
                                <string notr="true">background-color: #F44336; color: white;</string>
                            </property>
                            <layout class="QVBoxLayout" name="layoutError">
                                <property name="leftMargin">
                                    <number>20</number>
                                </property>
                                <property name="topMargin">
                                    <number>20</number>
                                </property>
                                <property name="rightMargin">
                                    <number>20</number>
                                </property>
                                <property name="bottomMargin">
                                    <number>20</number>
                                </property>
                                <item>
                                    <widget class="QLabel" name="labelError">
                                        <property name="text">
                                            <string>Kadi sio sahihi!</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                        <property name="wordWrap">
                                            <bool>true</bool>
                                        </property>
                                        <property name="font">
                                            <font>
                                                <pointsize>18</pointsize>
                                                <weight>75</weight>
                                                <bold>true</bold>
                                            </font>
                                        </property>
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                                                <horstretch>0</horstretch>
                                                <verstretch>0</verstretch>
                                            </sizepolicy>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </widget>

                        <!-- Success page -->
                        <widget class="QWidget" name="pageSuccess">
                            <property name="styleSheet">
                                <string notr="true">background-color: #4CAF50; color: white;</string>
                            </property>
                            <layout class="QVBoxLayout" name="layoutSuccess">
                                <property name="leftMargin">
                                    <number>20</number>
                                </property>
                                <property name="topMargin">
                                    <number>20</number>
                                </property>
                                <property name="rightMargin">
                                    <number>20</number>
                                </property>
                                <property name="bottomMargin">
                                    <number>20</number>
                                </property>
                                <item>
                                    <widget class="QLabel" name="labelSuccess">
                                        <property name="text">
                                            <string>Kadi imesomwa vizuri!</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                        <property name="wordWrap">
                                            <bool>true</bool>
                                        </property>
                                        <property name="font">
                                            <font>
                                                <pointsize>18</pointsize>
                                                <weight>75</weight>
                                                <bold>true</bold>
                                            </font>
                                        </property>
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                                                <horstretch>0</horstretch>
                                                <verstretch>0</verstretch>
                                            </sizepolicy>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </widget>

                    </widget>
                </item>
            </layout>
        </widget>
    </widget>
    <resources />
    <connections />
</ui>